settings/AdvancedSettings.cpp

```diff
@@ -272,7 +279,7 @@ void AdvancedSettings::loadFilters(QXmlStreamReader &xml)
  */
 void AdvancedSettings::loadMappings(QXmlStreamReader &xml, QHash<QString, QString> &mappings)
 {
-    mappings.clear();
+    // mappings.clear();
```

Comment:
Mappings should be cleared! Please add above mappings to `doc/advancedsettings.xml`.

Reply:
What is the reason? It’s just clear any preset mappings in AdvancedSettings.cpp
If `<videoCodecs></videoCodecs>` is empty - no mapping at all.

Reply:
The reason is that someone may not want those mappings. We currently have all default mappings included in `advancedsettings.xml`.

Reply:
OK. Deleted.

Reply:
Thanks :)